stable/cluster-overprovisioner/templates/deployments.yaml

```diff
@@ -16,6 +16,7 @@ metadata:
   name: "{{ $fullname }}-{{ .name }}"
   labels:
     app.kubernetes.io/name: {{ $name }}
+    app.kubernetes.io/fleetName: {{ .name }}
```

Comment:
I think it's a bit misleading using a label with `app.kubernetes.io/xxx` as this implies it's some sort of convention outside of this chart, which it isn't. Also `fleetName` is camelCase but I don't think this style is used in any other labels in k8s 🙂

So maybe just use something specific to this chart? Like `cluster-overprovisioner-name: {{ .name }}`?

Apart from that, a great addition!

Reply:
Sorry for the late reaction! (For some reason, Github didn't send me a notification..).

Naming things is always hard, and to be honest: I don't want to argue about that as long as it works and understandable. I changed the name to `cluster-overprovisioner-name`, please let me know what you think.

Thank you for the feedback!